src/hotspot/cpu/arm/universalUpcallHandle_arm.cpp

```diff
@@ -25,16 +25,12 @@
 #include "prims/universalUpcallHandler.hpp"
 #include "utilities/debug.hpp"
 
-address ProgrammableUpcallHandler::generate_upcall_stub(jobject rec, jobject jabi, jobject jlayout) {
-  Unimplemented();
-  return nullptr;
-}
-
-address ProgrammableUpcallHandler::generate_optimized_upcall_stub(jobject mh, Method* entry, jobject jabi, jobject jconv) {
+address ProgrammableUpcallHandler::generate_optimized_upcall_stub(jobject receiver, Method* entry,
```

Comment:
is the name "optimimzed" still carrying meaning in the new VM code?

Reply:
Not really, but I thought I'd save the renaming for: https://bugs.openjdk.java.net/browse/JDK-8275648

Reply:
Doing it here wouldn't be such a hassle though. Though, there's also OptimizedEntryBlob which requires renaming.

Reply:
That's ok - ship it as is - we can rename later.